### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by omarbabou**
In this PR, I Implemented the following steps:

   - Acquiring sustainable knowledge about code reviews.
   - Perform a code review for myself on a past project.
   - Fix linters error.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
```
Comment (omarbabou) [inline index.html:49]: Please remove the black spaces and the commented code so that it appears as clean code.

**Review by omarbabou — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re-review when you're done.

Thanks 😁

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio</h1>
+    </section>
+
+    <a
+      rel="noopener"
+      id="profile-link"
+      href="https://github.com/omarbabou"
+      target="_blank"
+    ></a>
```
Comment (Mre55) [inline index.html:27]: You can comment about this `<a>` tag currently it's not included inside any proper `tag`. Kindly, mention this to follow [semantic HTML tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.) rules.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @omarbabou,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._

**[REQUIRED]** Nice job adding one inline comment 👍. Kindly, add one more comment since this project require minimum of 2 inline comments. I add hint below for the other possible comment you can add.




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mre55) [reply index.html:49]: Nice job 👍

File: index.html (hunk 2 above)
Comment (omarbabou) [inline index.html:27]: Please use <a> tag inside an appropriate tag for the best practice of semantic HTML tags.

**Review by omarbabou — COMMENTED** [review]
# Required Changes ♻️ 

Check the comments under the review.

Thanks 😁

File: index.html (hunk 2 above)
Comment (omarbabou) [inline index.html:27]: Please use <a> tag inside an appropriate tag for the best practice of semantic HTML tags.

**Review by omarbabou — COMMENTED** [review]
# Required Changes ♻️ 

Check the comments under the review

Thanks 😁

File: index.html (hunk 2 above)
Comment (zieeco) [reply index.html:27]: **[OPTIONAL]**

- Great job on using the hint from the previous reviewer. You got that but there's a little issue with it. 
As of right now, your review  :point_right: **[Please use a tag inside an appropriate tag for the best practice of semantic HTML tags.]** :point_left: - is not explicit enough. It's still unclear to the code author as they don't really know what you mean. You should be able to tell the code author what appropriate tag to use to wrap the `a tag`. The word `tag` could refer to many tags in Html. You being able to specify the appropriate tag and give instances of usage will make your review process better and will make you a good code reviewer. 

_**weldone**_ and ***Keep rocking!***

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @omarbabou 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Impressive beginning 🔥 


## Optional suggestions
*Check the comment under the review*

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**